ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -66,6 +66,22 @@ ORK_AVAILABLE_DECL
  */
 - (void)passcodeViewControllerDidCancel:(UIViewController *)viewController;
 
+/*
```

Comment:
I've never read that, but would be interested in learning more.  Do you have a link reference to it being standard?  

To me, it makes sense in the case of `UITableViewDelegate` or `UITextFieldDelegate` to include themselves in the delegate method, considering there could be multiple of those, and the observer would need to know which is calling; however, I'm not understanding the case where you would need the view controller, since those rarely co-exist like that.